misc/ls-extensions/modules/bal-shell-service/src/main/java/io/ballerina/shell/service/ShellWrapper.java

```diff
@@ -101,6 +105,12 @@ public BalShellGetResultResponse getResult(String source) {
             } else if (shellCompilation.getExceptionStatus() == ExceptionStatus.SNIPPET_FAILED) {
                 throw new SnippetException();
             } else if (shellCompilation.getExceptionStatus() == ExceptionStatus.TREE_PARSER_FAILED) {
+                if (source.startsWith(COMMAND_PREFIX)) {
+                    List<Diagnostic> diagnostics = Collections.singletonList(Diagnostic.error(
+                            "Please note that Ballerina shell commands are not supported in here."));
```

Comment:
Shall we improve this message ?

IMO, it is not necessary to mention the ballerina shell here, what if we provide the error that we are getting instead of modifying it ?

As an example if user add `/abc` which is not a ballerina shell command and we will still provide a error message mentioning `Please note that Ballerina shell commands are not supported in here` which is not correct.

Reply:
Removed the modified error and provided the error generated by tree parser as it is. [`40558c7`](https://github.com/ballerina-platform/ballerina-lang/pull/37273/commits/40558c760c4979669408d0beed2760ea15600a44)